src/app/components/Pagination/index.tsx

```diff
@@ -46,16 +46,16 @@ interface PaginationProps {
 
 const visibilityToMediaQuery = (visibility: string) =>
   ({
-    [VISIBILITY.MOBILE_ONLY]: `display: none; @media (min-width: ${GROUP_2_MIN_WIDTH_BP}) and (max-width: ${GROUP_2_MAX_WIDTH_BP}) {
+    [VISIBILITY.MOBILE_ONLY]: `display: none; @media (min-width: ${GROUP_2_MIN_WIDTH_BP}rem) and (max-width: ${GROUP_2_MAX_WIDTH_BP}rem) {
```

Comment:
Do we need to update any tests? (If not, let's not hold up this PR, but ideally there would have been tests to cover this)

Reply:
What kind of tests would we usually want for CSS things. Would we want a snapshot test?

I could also add more to the cypress tests to make sure they cover a page with more numbers in the pagination

Reply:
Not sure! Because it's to do with the appearance of the ellipsis, probably a visual (Chromatic) test, or perhaps some type of unit test which could check the screenreader content (the presence of the ellipsis is supposed to announce that there's a separator), but let's just get this merged and not worry about the tests.

Reply:
When the ellipsis isn't visible, it is still present on the page, so I imagine it would still be present to a unit test even if it isn't visible on the page... 🤔 

I could look into adding a chromatic test (not done this before), or do another PR after this with a Cypress test as that will be able to check visibility rather than presence.